src/components/loginSignup/login.jsx

```diff
@@ -111,29 +109,35 @@ export default class Login extends Component {
                   <br></br>
 
                   <input
-                    className="txtbox"
+                    className={this.state.errors.password ? "txtboxRed" : "txtbox"}
```

Comment:
Ideally should compose classes...
`txtbox` should contain basic css for input
`borderRed` should just set border 
and when state is error then should set add extra class